### PR TITLE
docs(en/docks/upgrade/Air-Gapped-Upgrade-with-ks-installer.md): add d…

### PR DESCRIPTION
**Pull request by ruiyaoOps**
…ocs for Air-Gapped Upgrade with ks-installer.

docs(en/docks/upgrade/Air-Gapped-Upgrade-with-KubeKey.md): add docs for Air-Gapped Upgrade with KubeKey.md.

Signed-off-by: yaorui <yaorui@yunify.com>

### CODE REVIEW

**Comment by ruiyaoOps** [comment]
@FeynmanZhou  Please check my docs.

File: content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+# md5: 212091024dca02f8f34323e00f4be81d
+wget https://kubesphere-installer.pek3b.qingstor.com/offline/v3.0.0/kubesphere-all-v3.0.0-offline-linux-amd64.tar.gz
+```
+- Docker Registry
+
+You need to have a harbor or other Docker registry.
+
+{{< notice tip >}}
+
+You can [quick start yourself docker registry](https://kubesphere.com.cn/forum/d/2240-docker-registry)
```
Comment (pixiake) [inline content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md:32]: This is a Chinese article.

Comment (ruiyaoOps) [reply content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md:32]: I will change this to official documents.

**Comment by ruiyaoOps** [comment]
Signed-off-by: yaorui <yaorui@yunify.com>

**Pull request closed without merging**